plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/bean/BitstampBeanV1.java

```diff
@@ -80,21 +86,40 @@ public void setSubType(String field) {
     }
 
     @Override
-    public ImportedTransactionBean toImportedTransactionBean() {
+    public TransactionCluster toTransactionCluster() {
         validateCurrencyPair(amountCurrency, valueCurrency);
         if (!valueCurrency.equals(rateCurrency) || !valueCurrency.equals(feeCurrency)) {
             throw new DataValidationException(CURRENCY_EQUALITY_MESSAGE);
         }
+        List<ImportedTransactionBean> related;
+        if (ParserUtils.equalsToZero(fee)) {
+            related = Collections.emptyList();
+        } else {
+            related = List.of(
+                new FeeRebateImportedTransactionBean(
+                    FEE_UID_PART,
```

Comment:
if there is no uuid for the original TX, there's probably no need to synthesize one for the fee TX. This is also valid for other beans without tx uuid (Huobi, etc.)